package.json

```diff
@@ -27,28 +27,25 @@
   "scripts": {
     "test": "node test/test.js"
   },
-  "devDependencies": {
```

Comment:
This was due to a perhaps improper use of shrinkwrap. Shrinkwrap doesn't copy over devDependencies (since it is primarily intended for production), yet I was shrinkwrapping everything. Thus, dev installs weren't being filled with needed testing modules. It might make more sense to just drop the shrinkwrap file and return to devDependencies actually being meaningful.